### DISCUSSION TITLE
tdlib update

### DISCUSSION THREAD

**Discussion by cherryymerryy** [egramtel/tdsharp#239] [Q&A] [answered]
as far as I understand, the current version of TDLib is heavily updated in tdsharp, will there be an update?

**Comment by ForNeVeR** [comment]
Sure, why not. I'll add to my queue.

**Reply by cherryymerryy** [reply]
How soon can we expect an update?

**Reply by ForNeVeR** [reply, marked as answer]
When I have time, sorry. Perhaps during next week.

**Reply by cherryymerryy** [reply]
is there still no information about when the update will be?

**Reply by ForNeVeR** [reply]
All the information I have is provided above.
